featuretools/primitives/standard/transform_primitive.py

```diff
@@ -653,8 +653,8 @@ class Lag(TransformPrimitive):
         You can specify the number of periods to shift the values
 
         >>> lag_periods = Lag(periods=3)
-        >>> lag_periods(["hello", "world", "test", "foo", "bar"], pd.Series(pd.date_range(start="2020-01-01", periods=5, freq='D'))).tolist()
-        [nan, nan, nan, 'hello', 'world']
+        >>> lag_periods([True, False, False, True, True], pd.Series(pd.date_range(start="2020-01-01", periods=5, freq='D'))).tolist()
```

Comment:
Can we have a string lag and a boolean lag in the example?

Reply:
We could, but I'm not sure it's so simple: https://github.com/alteryx/featuretools/pull/2291#discussion_r975598983

If we added a string lag, the doctest wouldn't pass for all of the pandas version we currently support because of the `None` vs `nan` output difference. Unless we added some extra conversion to the output to make sure we always get the same missing value representation.

Reply:
Just want to point out, that once Woodwork gets initialized in the feature matrix we will always end up with the same missing value representation, but since that doesn't happen at the primitive function level, this difference in output will be present when calling it like we do in the doctest examples.

Reply:
Added comment: https://github.com/alteryx/featuretools/pull/2291/commits/b8e5cfe04986468df1b984613588a25d2a1019a6